app/src/main/java/com/eduard/caloriecounter/presentation/model/User.java

```diff
@@ -0,0 +1,33 @@
+package com.eduard.caloriecounter.presentation.model;
+
+public class User {
+
+    public String UserWeight,UserHeight,UserAge;
+    public static final int MAX_DECIMAL_DIGITS = 1;
+
```

Comment:
Also you may use default constants for age, weight, height and etc. This way app may perform calculation without waiting for user input.